ddr/lib/ddr-blobgen/java/genBinaryBlob.cpp

```diff
@@ -44,6 +44,11 @@
 #include <sys/types.h>
 #include <vector>
 
+/* For e2a_string() */
+#if defined(J9ZOS390) && !defined(OMR_EBCDIC) && defined(__open_xl__)
+#include "atoe.h"
+#endif /* defined(J9ZOS390) && !defined(OMR_EBCDIC) && defined(__open_xl__) */
```

Comment:
Please remove this unnecessary code, the include of `ddr/std/sstream.hpp`, and the using declaration for `stringstream`.